### ISSUE TITLE
portus 2.0.3 UI Access problem

### ISSUE THREAD

**Issue by hassanlhx** [SUSE/Portus#797]
In the network environment to access 10.0.0.1:3000

Viewing the network request with the following request causes the UI to access the existing problem

===========================

Remote Address:94.31.29.154:80

Request URL:http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css

Request Method:GET

===========================

Request URL:https://fonts.googleapis.com/css?family=Pacifico

Request Headers

Provisional headers are shown

Accept:text/css,*/*;q=0.1

Referer:http://10.0.0.1:3000/namespaces

User-Agent:Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko)

 Chrome/46.0.2490.80 Safari/537.36

===========================

Request URL:http://fonts.googleapis.com/css?family=Raleway:400,300,700,900

Request Headers

Provisional headers are shown

Accept:text/css,*/*;q=0.1

Referer:http://10.0.0.1:3000/namespaces

User-Agent:Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) 
Chrome/46.0.2490.80 Safari/537.36

These network requests are not appearing in the 2.0.0 version.How do I deal with this problem?

**Comment by mssola** [comment]
Could you detail what's your problem ? I'm not sure I understand.

**Comment by hassanlhx** [comment]
When I visit http://10.0.0.1:3000,After waiting for a long time in response to the page, I found that there was a request for http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css, which is why

**Comment by mssola** [comment]
Oh, I see. This has already been fixed in master. Portus will now serve these assets locally. Could you try to reproduce this in master ?

**Comment by hassanlhx** [comment]
There is no such problem in master.
I'm not sure I understand.

**Comment by mssola** [comment]
@hassanlhx This is something that we changed on master and that is going to be included in the `2.1` release. The change is basically that we will now serve all the assets locally (including the ones you're mentioning).

Closing this because it has already been fixed in master. That being said, if you have any more problems, feel free to add more comments here or send a message to our [Google groups](https://groups.google.com/forum/#!forum/portus-dev).
